tmp/lanelet2_extension/src/check_right_of_way.cpp

```diff
@@ -51,15 +52,15 @@ int main(int argc, char ** argv)
   lanelet::LaneletMapPtr map = lanelet::load(map_path, projector, &errors);
   for (auto && error : errors) std::cout << error << std::endl;
 
-  lanelet::traffic_rules::TrafficRulesPtr trafficRules =
+  lanelet::traffic_rules::TrafficRulesPtr traffic_rules =
     lanelet::traffic_rules::TrafficRulesFactory::create(
       lanelet::Locations::Germany, lanelet::Participants::Vehicle);
   lanelet::routing::RoutingGraphPtr routing_graph_ptr =
-    lanelet::routing::RoutingGraph::build(*map, *trafficRules);
+    lanelet::routing::RoutingGraph::build(*map, *traffic_rules);
 
   auto rows = map->regulatoryElementLayer  // filter elem whose Subtype is RightOfWay
               |
-              ranges::view::filter([](auto && elem) {
+              ranges::views::filter([](auto && elem) {
```

Comment:
Just out of curiosity, what was the warning of this part?

Reply:
Oh, it's this one.
https://github.com/autowarefoundation/autoware_common/pull/114#discussion_r1007863983